tests/unit/test_perturbation_metrics.py

```diff
@@ -0,0 +1,181 @@
+"""
+Test cases for perturbation metrics
+"""
+
+import itertools
+
+import pytest
+import numpy as np
+
+from armory.metrics import perturbation
+
+pytestmark = pytest.mark.unit
+
+
+def test_lp_norms():
```

Comment:
I think it'd make sense to also test `perturbation.element` for lp norms as well

Reply:
Yeah, that makes sense. I think it's probably unnecessary for all the norms, as the batch norms just combine results from the element norms, but it would be good to have something that tests them directly.

Reply:
Done.